src/radio/index.js

```diff
@@ -138,9 +138,9 @@ class Radio extends EventEmitter {
     }
 
     if (!this.order.some(u => u.equals(user))) return false;
-
     this.skips.add(user.id);
     this.checkSkips();
+
```

Comment:
<sub>super turbo ultra mega hyper nit: can we go back to the old spacing we had earlier? moving this newline above the `skips.add`. i like that one better</sup>

Reply:
🐇